### ISSUE TITLE
Add Rust v1.75.0 (Stable)

### ISSUE THREAD

**Issue by andria-dev** [codewars/runner#291]
- Name: Rust
- Version: 1.75.0 (Stable)
- Release Note/Changelog: https://releases.rs/docs/1.75.0/ (released on 28 Dec 2023)

---

:+1: reaction _might_ help to get this request prioritized.

**Comment by kazk** [comment]
Yeah, I've been wanting to add newer Rust, but it's been blocked by #250.

I just found out that `nextest` recently added experimental/partial support for a machine-readable output format compatible with libtest JSON.
```bash
NEXTEST_EXPERIMENTAL_LIBTEST_JSON=1 \
    cargo nextest run --message-format libtest-json --message-format-version 0.1
```

We need to try this and see how it works.

**Comment by hobovsky** [comment]
Rust has been updated to 1.86.
